jmin-jlink/Makefile

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2020, 2020 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+
+ifneq ($(MAKECMDGOALS),clean)
+ifndef JAVA_HOME
+${error JAVA_HOME must be exported to the environment or explicitly set in the make command}
+endif # JAVA_HOME
+endif # ($(MAKECMDGOALS),clean)
+
+PATH_TO_JDK   ?= ${JAVA_HOME}
+
+JAR           := $(PATH_TO_JDK)/bin/jar
+JAVAC         := $(PATH_TO_JDK)/bin/javac
+JLINK         := $(PATH_TO_JDK)/bin/jlink
```

Comment:
Why not just use `JAVA_HOME` instead of introducing the new macro `PATH_TO_JDK`?